services/site-scout-win/launch.sh

```diff
@@ -160,6 +160,9 @@ BUILD_SELECT_SCRIPT="import random;print(random.choice(str.split('${BUILD_TYPES}
 build="$(python -c "$BUILD_SELECT_SCRIPT")"
 # download build
 case $build in
+  beta-asan)
+    retry fuzzfetch -n build --asan --branch beta
```

Comment:
Do you not want `--fuzzing` builds?

Reply:
In this case I am trying to get closer to what we ship.